be/src/exprs/string_functions.cpp

```diff
@@ -2920,6 +2920,113 @@ StatusOr<ColumnPtr> StringFunctions::regexp_replace(FunctionContext* context, co
     return regexp_replace_general(context, options, columns);
 }
 
+struct ReplaceState {
+    bool only_null{false};
+    bool const_pattern{false};
+    std::string pattern;
+    bool const_repl{false};
+    std::string repl;
+};
```

Comment:
```suggestion
struct ReplaceState {
    bool only_null{false};
    bool const_pattern{false};
    bool const_repl{false};
    std::string pattern;
    std::string repl;
};
```
Adjust members' order to make the object smaller.